README.md

```diff
@@ -41,3 +41,18 @@ Owners of the Projects receive messages like this one:
 Messages like this one are sent to your Chat room:
 
 ![Example Chat message](example-chat-message.png?raw=true "Example Chat message")
+
+## Deploy as a Google Cloud Function
```

Comment:
Let's change the order of the topics and their titles a bit:

I was thinking something like the structure below may be better, can you change it please and we can evaluate the result?

```
# Zombie Projects Watcher

<let's bring the sample messages to this section and they dont have to be subtopics (i.e. ##)>

## Installation

### Dependencies and Config

    <text from the section: Installation>

    Add this part in this section: "Change `config.yaml` to you needs."

### Usage as a CLI command

    Remove this part from this section: "Change `config.yaml` to you needs " and ajust the text.

    <text from section: Usage>

### Usage as a Google Cloud Function

    <text from the section: Deploy as a Google Cloud Function>
```